lighthouse-core/test/audits/is-on-https-test.js

```diff
@@ -20,11 +20,11 @@ describe('Security: HTTPS audit', () => {
 
   it('fails when there is more than one insecure record', () => {
     return Audit.audit(getArtifacts([
-      {url: 'https://google.com/', scheme: 'https', domain: 'google.com'},
-      {url: 'http://insecure.com/image.jpeg', scheme: 'http', domain: 'insecure.com'},
-      {url: 'http://insecure.com/image.jpeg', scheme: 'http', domain: 'insecure.com'}, // should be de-duped
-      {url: 'http://insecure.com/image2.jpeg', scheme: 'http', domain: 'insecure.com'},
-      {url: 'https://google.com/', scheme: 'https', domain: 'google.com'},
+      {url: 'https://google.com/', parsedURL: {scheme: 'https', host: 'google.com'}},
```

Comment:
noisy changes to move off scheme and domain, which were just returning the `parsedURL` properties anyways